tools/breaking-change-detector/rules/rules_resource_schema.go

```diff
@@ -17,9 +17,21 @@ type ResourceSchemaRule struct {
 	isRuleBreak func(old, new map[string]*schema.Schema) []string
 }
 
-// ResourceSchemaRule is a list of ResourceInventoryRule
+// ResourceSchemaRules is a list of ResourceInventoryRule
 // guarding against provider breaking changes
-var ResourceSchemaRules = []ResourceSchemaRule{resourceSchemaRule_RemovingAField}
+var ResourceSchemaRules = []ResourceSchemaRule{resourceSchemaRule_RemovingAField, resourceSchemaRule_ChangingResourceIDFormat, resourceSchemaRule_ChangingImportIDFormat}
+
+var resourceSchemaRule_ChangingResourceIDFormat = ResourceSchemaRule{
+	name:       "Changing resource ID format",
+	definition: "Terraform uses resource ID to read resource state from the api. Modification of the ID format will break the ability to parse the IDs from any deployments.",
+	identifier: "resource-id",
+}
+
+var resourceSchemaRule_ChangingImportIDFormat = ResourceSchemaRule{
+	name:       "Changing resource ID import format",
+	definition: "Automation external to our provider may rely on importing resources with a certain format. Removal or modification of existing formats will break this automation.",
+	identifier: "resource-import-format",
+}
```

Comment:
Not in this PR, but I noticed the `resourceSchemaRule_RemovingAField` definition says "A removable of an field" instead of "a removal of a field"